api/server.py

```diff
@@ -48,27 +75,38 @@ def receive_edge_node_types():
     if not request.json:
         abort(400)
 
-mock_id = 1
 @app.route("/next-meta-paths", methods=["GET"])
 def send_next_metapaths_to_rate():
-    # TODO: Check if necessary information is in request object
-    global mock_id
-    batchsize = 5
-    paths = [{'id': i,
-              'path': ['Phenotype', 'HAS', 'Association', 'HAS', 'SNP', 'HAS', 'Phenotype'],
-              'rating': 0.5} for i in range(mock_id, mock_id + batchsize)]
-    mock_id += batchsize
-    # TODO: Call fitting method in active_learning
+    batch_size = 5
```

Comment:
As mentioned by @GittiHab: The batch_size could be also a parameter of the method and therefore of the call of the route.